fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/model/property/ElementBuilder.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.datagen.v1.model.property;
+
+import java.util.EnumMap;
+
+import com.google.common.base.Preconditions;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+import org.joml.Vector3d;
+
+import net.minecraft.util.math.Direction;
+
+/**
+ * Instantiate this class in order to provide any specific <code>elements</code> to a given block/item model JSON.
+ */
+public class ElementBuilder {
```

Comment:
final

Reply:
I assume this should also be the case for `FaceBuilder` and `OverrideBuilder`, correct?